bin/mtx_to_h5ad.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import scanpy as sc
+import pandas as pd
+import argparse
+
+
+def mtx_to_adata(
+    mtx_file: str, barcode_file: str, feature_file: str, sample: str, aligner: str, verbose: bool = False
+):
+
+    if verbose:
+        print("Reading in {}".format(mtx_file))
+
+    adata = sc.read_mtx(mtx_file)
+    if aligner == 'star': # for some reason star matrix comes transposed and doesn't fit when values are appended directly
+        adata = adata.transpose()
+    adata.obs_names = pd.read_csv(barcode_file, header=None)[0].values
+    adata.var_names = pd.read_csv(feature_file, header=None)[0].values
```

Comment:
these should be `tsv` files I think:
![image](https://user-images.githubusercontent.com/7051479/176473134-c40ccc16-8883-4f65-9d1d-4d9ddc5737e7.png)

Reply:
Indeed. Misses the "sep" argumen

Reply:
this one has been fixed